rpcserver.go

```diff
@@ -5563,17 +5563,19 @@ func handleTxFeeInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (
 // handleValidateAddress implements the validateaddress command.
 func handleValidateAddress(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
 	c := cmd.(*dcrjson.ValidateAddressCmd)
-
 	result := dcrjson.ValidateAddressChainResult{}
 	addr, err := dcrutil.DecodeAddress(c.Address)
 	if err != nil {
 		// Return the default value (false) for IsValid.
 		return result, nil
 	}
 
-	result.Address = addr.EncodeAddress()
-	result.IsValid = true
+	if !addr.IsForNet(s.server.chainParams) {
```

Comment:
Alright, after further discussion with the team, it seems that everyone would prefer it to not error in this case so it is consistent with both `dcrwallet` and other code in the ecosystem which expects that behavior.

Would you mind changing this back to:

```Go
result.IsValid = addr.IsForNet(s.server.chainParams)
```

Last time!